packages/worker/src/interceptors.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * Definitions for Activity interceptors.
+ *
+ * (The Worker also accepts Workflow interceptors but those are passed as module names)
+ *
+ * @module
+ */
+
+import { Next, Headers } from '@temporalio/workflow';
+import { Context as ActivityContext } from '@temporalio/activity';
+
+/** Input for ActivityInboundCallsInterceptor.execute */
+export interface ActivityExecuteInput {
+  readonly args: unknown[];
+  readonly headers: Headers;
+}
+
+/**
+ * Implement any of these methods to intercept Activity inbound calls
```

Comment:
I don't think it's immediately clear what `inbound` means. Arguably this whole thing could just be `ActivityCallsInterceptor` since it's not  clear to me if/why there would ever be an `Outbound` half.

But, even without that, would be good to elaborate slightly here and make it clear we're talking about calls that run activities (or, not sure what else later, but interact with existing activities maybe?)